src/com/goide/actions/fmt/GoImportsFileAction.java

```diff
@@ -64,7 +62,6 @@ public boolean doSomething(@NotNull PsiFile file,
     String filePath = virtualFile.getCanonicalPath();
     assert filePath != null;
 
-    GeneralCommandLine commandLine = new GeneralCommandLine();
     String sdkHome = GoSdkService.getInstance(project).getSdkHomePath(ModuleUtilCore.findModuleForPsiElement(file));
     if (StringUtil.isEmpty(sdkHome)) {
```

Comment:
GoExecutor will throw execution error if sdk is not set, don't need check it twice, you can catch the exception or just let executor write the error message in output console